language/src/modules/Language.rb

```diff
@@ -1431,6 +1402,79 @@ def fbiterm?
       Builtins.getenv("TERM") == "iterm"
     end
 
+    # Returns the command to make language settings persistent
+    #
+    # It is different depending on the stage, since for a not installed system it is needed to use
+    # the `systemd-firsrboot` tool, which does not work in a chroot
+    #
+    # @param locale [String]
+    #
+    # @return [Array<String>] an array containing the command to be executed
+    def store_locale_command(locale)
+      if Stage.initial
+        # do use --root option, running in chroot does not work
+        ["/usr/bin/systemd-firstboot", "-root", Installation.destdir, "--locale", locale]
```

Comment:
Invalid command parameter - in the past this was ```--root``` (not only ```-root```)

Very likely this is also the cause of e.g. https://openqa.opensuse.org/tests/883387#step/await_install/21

Reply:
Oops! 

It is a mistake made [during an update from code review](https://github.com/yast/yast-country/pull/206/commits/104c69cb123c71e1ce05046688f8a4f294a44b7a#diff-3b96e2c564f7d2e272c9457029f49ad5L1465). My fault :sweat_smile: I'll fix it ASAP.

Thank you!

Reply:
Already fixed and merged with https://github.com/yast/yast-country/pull/207

Reply:
I just see it :)

So, thank you so much! :)